api-2.2/src/main/java/org/openmrs/module/fhir/helper/ConditionHelperImpl2_2.java

```diff
@@ -65,6 +68,61 @@ public Condition createCondition(Condition condition) {
 		return generateFHIRCondition(openMrsCondition);
 	}
 
+	/**
+	 * @see org.openmrs.module.fhir.api.helper.ConditionHelper#searchConditionByPatient(java.lang.String)
+	 */
+	@Override
+	public List<Condition> searchConditionByPatient(String patientUuid) {
```

Comment:
When i first saw searchConditionByPatient, i thought it takes a name. Can we rename this to searchConditionByPatientUuid?